pep-0480.txt

```diff
@@ -341,14 +345,15 @@ distributions, and prevents MITM attacks on usernames and passwords.
 __ https://github.com/pypa/twine
 
 
-Distutils
----------
+Distlib
+-------
 
-`Distutils`__ MAY be modified to sign metadata and to upload signed distributions
-to PyPI.  Distutils comes packaged with CPython and is the most widely-used
-tool for uploading distributions to PyPI.
+`Distlib`__ MAY be modified to sign metadata and to upload signed
+distributions to PyPI.  Distlib is a library which implements
+low-level functions that relate to packaging and publication of
+Python software, including uploading distributions to PyPI.
```

Comment:
It feels very much like you think distlib and distutils are similar. That isn't true - setuptools is much more likely to be what you mean if you're thinking of distutils-equivalents...

Reply:
I made this edit months ago and so past Sumana was probably distracted or had a worse understanding than present-day Sumana. Past Sumana, come on, help me out here!

I'll fix this, Paul; thanks.